kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -67,7 +67,7 @@
         </div>
       </div>
 
-      <div v-if="!isTopicIdSet && bookmarks.length && !showBookmarks">
+      <div v-if="!isTopicIdSet && bookmarks.length && !showBookmarks && !showSearch">
```

Comment:
[Non blocking] The render conditions of this component are getting each time more complex, it would be good if in the future we can find a way of making this more easy to follow. Perhaps breaking down this component into more specific use cases? like `BookmarksResourceSelection`, `SearchResourceSelection`, etc.

Reply:
Oh yes for sure! All of the changes here to ResourceSelection are basically for demonstration purposes - although @marcellamaki and I are drafting an issue which will hopefully refactor this component quite a bit